packages/core/src/feed.ts

```diff
@@ -1,20 +1,53 @@
-import type { StreamMetadata, StreamState } from '@ceramicnetwork/common'
+import { CommitType, LogEntry, StreamMetadata, StreamState } from '@ceramicnetwork/common'
 import { Subject, type Observable } from 'rxjs'
 import { CommitID } from '@ceramicnetwork/streamid'
 import { StreamUtils } from '@ceramicnetwork/common'
 
+/**
+ * Describes all event types
+ */
+export enum EventType {
+  CREATE = 0,
+  ANCHOR = 1,
+  UPDATE = 2,
+}
+
+/**
+ * Checks last log in stream state and categorizes the event accordingly
+ * @param log
+ * @returns EventType
+ */
+function parseCommitType(log: LogEntry[]): EventType {
+  const length = log.length
+
+  if (length == 1) return EventType.CREATE
+
+  switch (log[length - 1].type) {
+    case CommitType.ANCHOR:
```

Comment:
why not just use the CommitType enum (and make a change to the names of the values in that enum to match the current naming convention)?  Why have two enums and map between the two of them?